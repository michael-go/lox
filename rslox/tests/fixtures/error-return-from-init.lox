class CoffeeMaker {
  init(coffee) {
     this.coffee = coffee;
     return coffee;
  }
}

print CoffeeMaker("coffee and chicory");