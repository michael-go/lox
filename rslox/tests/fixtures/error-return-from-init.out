# exit code: 65
# stdout:

# stderr:
[line 4] Error at 'return': Can't return a value from an initializer.
[line 5] Error at '}': Expect expression.

